game-core/src/main/java/games/strategy/engine/framework/HeadlessAutoSaveType.java

```diff
@@ -7,21 +7,21 @@
  * The types of auto-saves that can be loaded by a headless game server.
  */
 public enum HeadlessAutoSaveType {
-  DEFAULT(AutoSaveFileUtils.getHeadlessAutoSaveFile()),
+  DEFAULT(new HeadlessAutoSaveFileUtils().getHeadlessAutoSaveFile()),
```

Comment:
It is a bit of a code smell for new objects to be created without state to call a no-param method, does seem like it could be a static method call, ie: `SaveFileUtils.getHeadlessAutoSaveFile()`

I think probably more the enum might need to be flipped. Instead of having  path variable, have the caller compute the path. EG: `HeadlessAutoSaveFileUtils.getSaveFilePath(HeadlessAutoSave.ODD_ROUND)`, in that case this enum would be a simple enum without the path variable.